modules/ossm-jaeger-architecture.adoc

```diff
@@ -0,0 +1,21 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-jaeger.adoc
+////
+
+[id="ossm-jaeger-architecture_{context}"]
+= Jaeger architecture
+
+Jaeger is made up of several components that work together to collect, store, and display tracing data.
+
+* *Jaeger Client* (Tracer, Reporter, instrumented application, client libraries)- Jaeger clients are language specific implementations of the OpenTracing API. They can be used to instrument applications for distributed tracing either manually or with a variety of existing open source frameworks, such as Flask, Dropwizard, gRPC, and many more, that are already integrated with OpenTracing.
+
+* *Jaeger Agent* (Server Queue, Processor Workers) - The Jaeger agent is a network daemon that listens for spans sent over User Datagram Protocol (UDP), which it batches and sends to the collector. The agent is meant to be placed on the same host as the instrumented application. This is typically accomplished by having one agent per bare metal for traditional applications, or as a sidecar in container environments like Kubernetes.
+
+* *Jaeger Collector* (Queue, Workers) - Similar to the Agent, the Collector is able to receive spans and place them in an internal queue for processing. This allows the collector to return immediately to the client/agent instead of waiting for the span to make its way to the storage.
+
+* *Storage* (Data Store) - Collectors require a persistent storage backend. Jaeger’s storage is a pluggable component which uses Elasticsearch for storage. Note that for this release, the only supported storage is Elasticsearch.
```

Comment:
> Jaeger’s storage is a pluggable component which uses Elasticsearch for storage.

This sounds a bit strange. I would just leave this phrase out, or use something like: "Jaeger has a pluggable mechanism for the span storage. Note that for this release, ..."